dotcom-rendering/scripts/jest/setup.ts

```diff
@@ -14,6 +14,8 @@ const windowGuardianConfig = {
 		browserId: 'jest-browser-id',
 		pageViewId: 'jest-page-view-id',
 	},
+	tests: {
```

Comment:
This is required in order to satisfy tests and, as this is a required field on the config object, it seems reasonable to extend this mock to include it. The config object has been typecast as windowGuardianConfig which is why TS hasn't complained before without this required field.